### DISCUSSION TITLE
security onion on home network, questions

### DISCUSSION THREAD

**Discussion by CapaBE** [Security-Onion-Solutions/securityonion#5179] [Unsupported Versions] [answered]
Hello,  
I have some questions about security onion that I would like to use on my home network.  
I would install it (standalone) on a laptop that I don't use anymore.

From what I have read, you need 2 nic's on the laptop, 1 for management and 1 for sniffing.  
I just have 1 nic but I can attach a USB nic for management so I can use the internal (better) nic for sniffing.

I have a question about the sniffing part.  I have more than 1 switch in my home network and most of the connections are wired to a non-managed 16 port switch.  I also have 2 smaller managed switches in other parts of the house.  
Does that mean that I can only monitor network traffic that is going through 1 of the smart switches since I need to connect my laptop to a span port?  Off course I would like to monitor all traffic and not only 1 of the 2 smart switches.

Do I have more possibilities with a network TAP maybe?
Can someone give me some advice on how I can do this in the best way ?

Thanks in advance

**Comment by gcsrellis** [comment, marked as answer]
I would span the closest point of entry on your network.  So if you have your router connecting to a managed switch, that's what I'd span.  Then trunk your other switches off that one managed switch.  This would give you the ability to span all incoming and outgoing traffic from your network.

**Reply by CapaBE** [reply]
thnx, I will have to change the layout of the network a bit then but it sounds logic  :)
